src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
I think Link is too generic. This component deals with Web Linking. It's a very specific concept. When you say link, people usually think of URI, which is not what this is about.